src/main.cpp

```diff
@@ -32,6 +32,13 @@
 
 #include "algo.h"
 
+#if defined(__APPLE__)
+    #include <unistd.h>             // Required for: usleep()
```

Comment:
usleep is never used

Reply:
I thought I had removed that but it turns out this was after committing.